src/legacy/core_plugins/vis_type_timeseries/public/components/vis_editor_visualization.js

```diff
@@ -70,12 +70,12 @@ class VisEditorVisualizationUI extends Component {
       return;
     }
 
-    const { timeRange, appState, savedObj, onDataChange } = this.props;
+    const { timeRange, savedObj, onDataChange } = this.props;
 
     this._handler = await embeddables.getEmbeddableFactory('visualization').createFromObject(savedObj, {
       vis: {},
       timeRange: timeRange,
-      filters: appState ? appState.filters || [] : [],
+      filters: [],
```

Comment:
:information_source: `appState` was never passed into that component at all, so this never evaluated to any filters. I can also not see any use-case where there would be filters on the `appState` for TSVB, since it doesn't have a filter bar in the editor and on dashboard it's coming not in via the `appState` (and testing it showed that it all works on dashboard - please verify during testing)